src/Tribe/Rewrite.php

```diff
@@ -1024,7 +1024,7 @@ public function get_clean_url( $url, $force = false ) {
 			return home_url();
 		}
 
-		$clean = $this->get_canonical_url( add_query_arg( $parsed_vars, home_url() ), $force );
+		$clean = $this->get_canonical_url( add_query_arg( $parsed_vars, home_url( '/' ) ), $force );
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :warning: Line exceeds 80 characters; contains 100 characters

<sub>hash: cf46c6100dd44b49f031c7d3f4b5b41f</sub>